<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWidget</class>
 <widget class="QWidget" name="LoggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="filePathLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browsePushButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="appendCheckBox">
     <property name="text">
      <string>Append</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>&lt;font color=red&gt;00:00:00&lt;/font&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="startPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filePathLineEdit</tabstop>
  <tabstop>browsePushButton</tabstop>
  <tabstop>appendCheckBox</tabstop>
  <tabstop>startPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
